Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,50 @@
+
+#include <iostream>
+#include <string>
+#include "Person.h"
+
+Person::Person() : name_(""), age_(nullptr) {}
+
+Person::~Person() {
+    delete this->age_;
```

Comment:
With how we have private variables to have a trailing underscore. We really don't need the `this` keyword